content/applications/sales/point_of_sale/overview/getting_started.rst

```diff
@@ -2,148 +2,105 @@
+From the dashboard, click on :guilabel:`New Session` to start a POS session, or :guilabel:`Resume`
+if the session was already open, and reach the POS interface.
 
-Your first order
-----------------
+.. image:: getting_started/pos-interface.png
+   :align: center
+   :alt: Point of sale dashboard.
 
-Open a new session from the dashboard by clicking on *New Session*.
-
-.. image:: getting_started/getting_started_04.png
-    :align: center
-    :alt: Launching a new session.
-
-After the loading screen, you arrive on the PoS interface.
+.. note::
+   Multiple users can be connected to the same session at the same time. However, the session can
+   only be open once in the same browser.
 
-.. image:: getting_started/getting_started_05.png
-    :align: center
-    :alt: Point of sale dashboard.
+Click on products to add them to the cart. To change the **quantity**, click on :guilabel:`Qty` and
+enter the number of products using the keypad. To add a **discount** or modify the product
+**price**, click respectively on :guilabel:`Disc` or :guilabel:`Price` and enter the amounts using
+the keypad.
 
-Once an order is completed, you can register the payment. All the
-available payment methods appear on the left of the screen. Select the payment
-method and enter the received amount. Then, you can validate the
-payment.
+Once an order is completed, proceed to checkout by clicking on :guilabel:`Payment`. Select the
+**payment method**, enter the received amount, and click on :guilabel:`validate`. Click on
+:guilabel:`New Order` to move on to the next customer.
 
 .. tip::
-   You can use both `,` and `.` on your keyboard as decimal separators.
+   - You can use both `,` and `.` on your keyboard as decimal separators.
+   - **Cash** is selected by default if you enter the amount without choosing a payment method.
 
 Return and refund products
 --------------------------
 
-Having a well-thought-out return policy is key to keep customers satisfied and make the process 
-of accepting returns and refunds easy for you.
-
-To do that, from the PoS interface, select the product and quantity (with the +/- button) that the 
-customer wants to return. For multiple products, repeat the process individually.
+To return a product and make a refund:
 
-.. image:: getting_started/getting_started_06.png
-    :align: center
-    :alt: Refunding a product with the point of sale app.
-
-When on the payment interface, the total is negative. To end the refund,
-process the payment and validate it.
-
-.. image:: getting_started/getting_started_07.png
-    :align: center
-    :alt: Giving back money to customer in Odoo Point of Sale.
+#. **Open a session** and select the returned product;
+#. Click on :menuselection:`Qty --> +/-` to enter the quantity of returned products;
+#. Once the order is completed, click on :guilabel:`Payment` to proceed to the refund;
+#. Click on :menuselection:`Validate --> New Order` to move on to the next customer.
 
 Close the PoS session
```

Comment:
```suggestion
Close the PoS session
```
Here it is spilled PoS, whereas the rest (or most) of the document is spelled POS. It is better to select one or the other for consistency